src/web/components/ClientSideTokenGeneration/CstgAddDomainDialog.tsx

```diff
@@ -46,27 +45,16 @@ function CstgAddDomainDialog({
         message: 'The domains entered already exist.',
       });
     } else {
-      const invalidDomains: string[] = [];
-      let allValid = true;
-      uniqueDomains.forEach((newDomain) => {
-        if (!isValidDomain(newDomain)) {
-          invalidDomains.push(newDomain);
-          allValid = false;
-        }
-      });
-      if (!allValid) {
+      // filter for uniqueness (e.g. 2 different domains entered could have the same root-level domain)
```

Comment:
I find this comment/naming a bit confusing because we already have `const uniqueDomains` above on line 39. Maybe we could think of a better name to avoid confusion between uniqueness and deduped?